Translations/WinMerge/Dutch.po

```diff
@@ -2,17 +2,17 @@
 # Released under the "GNU General Public License"
 #
 # Translators:
-# * Ronald Schaap <ronald at schaap.demon.nl>
-# * Thomas De Rocker, 2023
```

Comment:
@RockyTDR 
The `*` at the beginning is necessary when generating TranslationsStatus.md etc. If you remove this, your name will be
It disappears from https://github.com/WinMerge/winmerge/blob/master/Translations/TranslationsStatus.md.
For this I added `*` in commit https://github.com/WinMerge/winmerge/commit/3d1092cd32b71ec8081f8604daaba4b70cac6f6d.

Reply:
Oh, thank you! Transifex automatically removes the asterisk before translators... I'll keep it in mind and add it manually next time!